src/Bridges/bridge_optimizer.jl

```diff
@@ -453,8 +457,16 @@ end
 function MOI.supports_incremental_interface(b::AbstractBridgeOptimizer)
     return MOI.supports_incremental_interface(b.model)
 end
-function MOI.Utilities.final_touch(uf::AbstractBridgeOptimizer, index_map)
-    return MOI.Utilities.final_touch(uf.model, index_map)
+
+function final_touch(b::AbstractBridgeOptimizer)
```

Comment:
It's nice to be able to final touch the bridges without the rest. That's a good argument to have a `MOI.Bridges.final_touch`.